test/test_feature_library.py

```diff
@@ -115,33 +115,59 @@ def test_pde_library_bad_parameters(params):
 @pytest.mark.parametrize(
     "params",
     [
-        dict(spatiotemporal_grid=range(10), p=-1),
-        dict(spatiotemporal_grid=range(10), H_xt=-1),
-        dict(spatiotemporal_grid=range(10), H_xt=11),
-        dict(spatiotemporal_grid=range(10), K=-1),
+        dict(
+            spatiotemporal_grid=range(10),
+            p=-1,
+            library_functions=[lambda x: x, lambda x: x**2, lambda x: 0 * x],
+        ),
+        dict(
+            spatiotemporal_grid=range(10),
+            H_xt=-1,
+            library_functions=[lambda x: x, lambda x: x**2, lambda x: 0 * x],
+        ),
+        dict(
+            spatiotemporal_grid=range(10),
+            H_xt=11,
+            library_functions=[lambda x: x, lambda x: x**2, lambda x: 0 * x],
+        ),
+        dict(
+            spatiotemporal_grid=range(10),
+            K=-1,
+            library_functions=[lambda x: x, lambda x: x**2, lambda x: 0 * x],
```

Comment:
Previously, the `library_functions=[lambda x: x, lambda x: x**2, lambda x: 0 * x] ` was specified in the `test_weak_pde_library_bad_parameters` in an odd way rather than in the `@pytest.mark.parametrize` part, and all the test used the same `library_functions`. Since I wanted to add the case with `library_functions=[]`, I restructured, which made more sense anyways.